src/cli.ts

```diff
@@ -181,7 +188,10 @@ async function runWithPaths(paths: Array<string>, options: CLIOptions): Promise<
 
   async function processFile(path: string): Promise<void> {
     let extension = path.endsWith('.coffee.md') ? '.coffee.md' : extname(path);
-    let outputPath = join(dirname(path), basename(path, extension)) + '.js';
+    if (options.output && !(await exists(options.output))) {
+      await mkdir(normalize(options.output));
```

Comment:
This will only create the root directory, but nothing else. This will work well enough if converting a single file, but for converting a directory structure the `writeFile` call in `processFile` will fail for any files in a nested directory.

It would make this much easier to add to `cli_test.ts` to make sure that the cases you care about work as expected.